yogstation/code/modules/jobs/job_types/lawyer.dm

```diff
@@ -0,0 +1,2 @@
+/datum/job/lawyer
+	minimal_character_age = 24
```

Comment:
You need to be older to be a lawyer than to be the head of security or head of personnel?

Reply:
I was trying to make these values as low as feasibly possible. They don't have to be *that* high to make IC age not immersion-breaking .